modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,64 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter that helps you conserve resources by limiting the number of times the Mule runtime engine (Mule) executes messages that result in delivery failures. You can add a redelivery policy to some connectors that contain flow sources, such as the xref:connectors::salesforce/salesforce-connector-reference.adoc[Salesforce], xref:connectors::sap/sap-connector-reference.adoc[SAP], and xref:connectors::vm/vm-reference.adoc[VM] connectors. To see whether a connector can use a redelivery policy, see the reference document for that connector.
```

Comment:
remove `To see whether a connector can use a redelivery policy, see the reference document for that connector.`. The support of redelivery policies is added by the runtime in all sources, regardless of the source